examples/openstack/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```

Comment:
It appears that Bundler already handles that problem.

I ran `bundle install --gemfile examples/docker_provider/Gemfile` from the project's root directory and Bundler resolved `../../` as the project's root directory.